test/functional/Liquidation.js

```diff
@@ -0,0 +1,96 @@
+const { expect } = require("chai")
+const { ethers, getNamedAccounts, deployments } = require("hardhat")
+
+describe("Liquidation functional tests", async () => {
```

Comment:
It'd be nice to nest `describe` calls, like so:

```javascript
describe("Liquidation", async () => {
    describe("someLiqFunc", async () => {
        context("When called with foobar", async () => {
            it("Passes", async () => {
                /* ... */
            });
        });
    });
});
```

Reply:
I don't know if that's any better though. The output would require reading the extra "Passes" line to see what the expected result of that test case is.